transbank/webpay/webpayplus/transaction.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from transbank.webpay.webpayplus.transaction_create_response import TransactionCreateResponse
+from transbank.webpay.webpayplus.webpayplus import *
+
+
+class Transaction:
+    CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    REFUND_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}/refund'
+    GET_TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}'
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def create(cls, buy_order, session_id, amount,
+               return_url, options=None):
+        commerce_code = None
+        api_key = None
+        base_url = None
+        if options is None:
+            commerce_code = WebpayPlus.commerce_code()
+            api_key = WebpayPlus.api_key()
+            base_url = WebpayPlus.integration_type_url()
+        else:
+            commerce_code = options.commerce_code
+            api_key = options.api_key
+            # WebpayPlus.integration_type_url is a setter method that works with
+            # a key which is supplied by option.integration_type
+            # This is not so intuitive as "getter / setter" style from java / PHP
+            WebpayPlus.integration_type_url = options.integration_type  # setter method
+            base_url = WebpayPlus.integration_type_url()  # Class method
```

Comment:
Please don't add that kind of comments. You shouldn't need to know if it's a setter method or not when reading code. And I do understand how `@property` setters/getters work. The original problem with this code is not about understanding how python works (or whether something is or isn't a setter method). It's about surprising the reader of this code (which is still going to be surprised with or without the comments) and the user of the SDK (who will never see these comments).